core/services/cable_guy/html/index.html

```diff
@@ -2,9 +2,10 @@
 <html lang="en">
 
 <head>
-  <!-- Required meta tags -->
-  <meta charset="utf-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
+   <title>Cable-guy</title>
+   <!-- Required meta tags -->
+   <meta charset="utf-8">
+   <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
```

Comment:
wait, the linter was not checking here ?

Reply:
good catch, let me check

Reply:
it turns out that it catches indentation errors, but not always. I opened https://github.com/bluerobotics/companion-docker/issues/189

Reply:
How is that possible ? We have not enabled cache under github actions for this

Reply:
I'm guessing it is too permissive to what is "acceptable" indentation.
If you try the current config with https://github.com/bluerobotics/companion-docker/blob/5e4dd905560fbebcefc88c90b3c539065bf34ed3/core/services/cable_guy/html/index.html
it seems to fail properly.